checks/run_probes.go

```diff
@@ -23,20 +23,15 @@ import (
 	"github.com/ossf/scorecard/v4/probes/zrunner"
 )
 
-// evaluateProbes runs the probes in probesToRun and logs its findings.
-func evaluateProbes(c *checker.CheckRequest, rawResults *checker.RawResults,
+// evaluateProbes runs the probes in probesToRun.
+func evaluateProbes(rawResults *checker.RawResults,
```

Comment:
without logging, all the callers could probably call zRunner directly now?

Reply:
will look into, I think you're right